channels/views/moderators_test.py

```diff
@@ -12,7 +12,7 @@
 
 def test_list_moderators(client, private_channel_and_contributor, staff_user):
     """
-    List moderators in a channel
+    List moderators in a channel as a logged in user
```

Comment:
Minor nitpick: change to something like `List moderators in a channel as a logged in contributor`?